### ISSUE TITLE
Auto double cast won't trigger [Question/Possible Bug]

### ISSUE THREAD

**Issue by LyonV** [erbkaiser/FrozenCookies#170]
I'm honestly not sure if this is a bug or if I am doing something wrong, but for some reason the auto double cast does not seem to want to trigger. I have read the read-me and I believe I meet the requirements. 
I have level-1 wizard towers at 100 mana. I turned on the auto double cast feature and it began casting haggler's charm as expected. It finally managed to get a CF+BS lined up as the next two spells and paused. I also have the DH aura. Now, to my understanding, the system should be waiting for a natural frenzy/DH+BS, once it detects this, it should execute the double cast. Is this correct? Because, I have seen both of those combos pass by several times now, yet the auto cast is not getting triggered. Am I missing something, or is this not expected behavior?

**Comment by erbkaiser** [comment]
If Reaper of Fields is your aura, it will need an active Frenzy AND a Dragon Harvest, as well as the BS
The assumption is that by setting Reaper, you want the maximum combo possible.

If Reaper is not the active aura, it wil need a Frenzy OR Harvest, and of course the BS

The Frenzy + DH applies to smart FTHOF auto-casting as well, if Reaper is the aura.

**Comment by LyonV** [comment]
I see, so it's just a matter of waiting longer for the perfect combo with that aura enabled. 
So it's possible to get the harvest without the aura? I didn't know this. I guess that's why I misunderstood the readme.
Thanks for replying so quickly!

**Comment by LyonV** [comment]
Also, one more question, sorry. How long would you say it might take on average to get the required combo when using the DH aura? I mean, is this something that should happen in a day, or is it probably going to take a while longer?

Edit: For clarification, I do have all upgrades pertaining to GCs. Including the heavenly ones.

**Comment by erbkaiser** [comment]
If you have any ideas on how to make the readme more clear, please let me know, either with a bug report or a pull request.
I did my best to explain everything but it's possible things can be reworded for clarity.

As for the auras: there is the dragon aura that has 10% of all other auras -- I think it's called Reality Bending . If you have that set, you can occasionally get a DH as well.

Golden cookie probablities are dependent on a lot of factors but some quick math (may be slightly incorrect, I'm not going to look it all up in detail):
Your base chance for a Frenzy is 10%, with another 0.5% additional chance if Dragonflight or RB is your aura
Dragon Harvest has a flat 15% if Reaper is the aura, and a 1,5% chance if RB is.
So if you have Reaper of Fields your chance of a Frenzy/DH stack is about 0,575% and if you have RB, it is about 0,075%.

Building Special has about a 25% as well, and the combo will only fire if it has enough time left in the Frenzy and/or DH when the BS drops.

TLDR: it can take a LOT of time, depending on how lucky you are, but if you do manage to pull the combo off you have a 105% boost from the Frenzy and DH combo alone, with the two BS on top of that, so it can definitely be worth it.

Personally, I don't use Reaper as the Aura, but I do often have RB stacked (since it also helps with other effects) so I do have the occasional DH 15x instead of the Frenzy 7x.
